src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.MonoCrossAOT.UnixFilePermissions.xml

```diff
@@ -0,0 +1,5 @@
+<FileList>
+    <File Path="tools/mono-aot-cross" Permission="755" />
+    <File Path="tools/opt" Permission="755" />
+    <File Path="tools/llc" Permission="755" />
```

Comment:
Do these need .exe for windows?

Reply:
I don't think `UnixFilePermissions.xml` is needed at all on Windows.